tests/tamr_client/utils.py

```diff
@@ -13,7 +13,7 @@ def load_json(path: Union[str, Path]):
 
 
 def session():
-    auth = tc.UsernamePasswordAuth("admin", "dt")
+    auth = tc.UsernamePasswordAuth("username", "password")
```

Comment:
You should undo this change (i.e. `git rebase -i master` and edit the commit where this change was made so that its not included) and then:

```sh
git checkout master
git pull Datatamer master
git checkout beta-update-dataset
git rebase master
git push -f skalish beta-update-dataset
```
since this change is already on master.

Reply:
I think this is fixed now.